app/Presenters/AssetPresenter.php

```diff
@@ -562,6 +562,36 @@ public function dynamicWarrantyUrl()
         return $url;
     }
 
+    /**
+     * Used to take user created Support URL and dynamically fill in the needed values per asset
+     * @return string
+     */
+    public function dynamicSupportUrl()
+    {
```

Comment:
I think we can probably combine these two, no? It's effectively the same code repeated.